webpack.config.js

```diff
@@ -3,6 +3,13 @@ const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const VueLoaderPlugin = require("vue-loader/lib/plugin");
 const CopyPlugin = require("copy-webpack-plugin");
+const packageJson = require("./package.json");
+const version = packageJson.version || 0;
+const versionPrefix = "alpha-";
```

Comment:
```suggestion
const versionPrefix = "alpha";
```

Let's delegate adding the hyphen to the use site below in `APP_VERSION`.

Reply:
Right sure